src/app/(private)/admin-registration-applications/page.tsx

```diff
@@ -0,0 +1,58 @@
+'use client'
+import PrivateRoute from '@/app/hocs/isAuth'
+import SelectStatus from '@/components/adminRegistrationApplications/selectStatus'
+import { useEffect, useState } from 'react'
+import Pagination from '@/components/adminRegistrationApplications/pagination'
+import { postAdminRegistrationApplications } from '@/modules/adminRegistrationApplications/service'
+import { useTranslation } from 'react-i18next'
+import DataTable from '@/components/adminRegistrationApplications/dataTable'
+import { SortingState } from '@tanstack/react-table'
+interface AdminRegistrationState {
+  content: []
+  totalPageCount: number
+}
+const Page = ({ searchParams }: { searchParams: any }) => {
+  const [selectStatus, setSelectStatus] = useState([])
+  const [adminRegistration, setAdminRegistration] =
+    useState<AdminRegistrationState>({ content: [], totalPageCount: 0 })
+  const [isLoading, setIsLoading] = useState(true)
```

Comment:
isLoading anlamı 'yükleniyor mu'. Bunun için isLoading false olmalıdır, true olduğunda loading çalışıyor olmalıdır. Etkilenen tüm alanlar düzeltilmelidir